docs/01-app/01-getting-started/01-installation.mdx

```diff
@@ -0,0 +1,353 @@
+---
+title: How to set up a Next.js project
+nav_title: Installation
+description: Learn how to create a new Next.js application with `create-next-app`, and set up TypeScript, ESLint, and Module Path Aliases.
+---
+
+{/* The content of this doc is shared between the app and pages router. You can use the `<PagesOnly>Content</PagesOnly>` component to add content that is specific to the Pages Router. Any shared content should not be wrapped in a component. */}
+
+## System Requirements
```

Comment:
This uses title case, whereas the rest of the page is sentence case (as it should be)
```suggestion
## System requirements
```